packages/ag-charts-community/src/chart/series/cartesian/util.ts

```diff
@@ -68,6 +73,12 @@ export function calculateSegments(
             let startPosition = scale.convert(start ?? startFallback) - offset;
             let stopPosition = scale.convert(stop ?? stopFallback) + 2 * offset;
 
+            const invalidStart = start != null && isNaN(startPosition);
+            const invalidStop = stop != null && isNaN(stopPosition);
+            if (invalidStart || invalidStop) {
+                continue;
+            }
+
             if (isNaN(startPosition)) startPosition = getDefaultStart();
             if (isNaN(stopPosition)) stopPosition = getDefaultStop();
```

Comment:
Could these be `startPosition ??= getDefaultStart()` since you checked `isNaN` above so it should either be `null` or a number now?

Reply:
discussed offline